librecad/src/lib/engine/rs_entitycontainer.cpp

```diff
@@ -229,6 +229,14 @@ bool RS_EntityContainer::toggleSelected() {
 }
 
 
+void RS_EntityContainer::setHighlighted(bool on)
```

Comment:
There is the overridden function of the `RS_Entity::setHighlighted` function.
Without this, you cannot highlight groups of entities, like `RS_Polyline`, `RS_Dimension`, etc.